src/main/java/com/uber/jenkins/phabricator/PhabricatorNotifier.java

```diff
@@ -188,21 +195,15 @@ private CoverageProvider getUberallsCoverage(AbstractBuild build, BuildListener
         }
 
         Logger logger = new Logger(listener.getLogger());
-
-        Provider<CoverageProvider> provider = new BaseProvider<CoverageProvider>(
+        InstanceProvider<CoverageProvider> provider = new InstanceProvider<CoverageProvider>(
```

Comment:
How do I make a generic static method? I'm getting "intstanceprovider.this cannot be referenced from a static context"